addons/pos_restaurant/static/tests/tours/TipScreen.tour.js

```diff
@@ -1,157 +1,156 @@
+            // order 4
+            ProductScreen.addOrderline("Coca-Cola", "4", "2"),
+            ProductScreen.totalAmountIs("8.0"),
+            FloorScreen.backToFloor(),
+            FloorScreen.orderCountSyncedInTableIs("5", "4"),
+            Chrome.clickMenuButton(),
+            Chrome.clickTicketButton(),
+            TicketScreen.nthRowContains("4", "Tipping"),
 
-        // Create without syncing the draft.
-        // order 3
-        FloorScreen.do.clickTable("5");
-        ProductScreen.exec.addOrderline("Minute Maid", "3", "2");
-        ProductScreen.check.totalAmountIs("6.0");
-        ProductScreen.do.clickPayButton();
-        PaymentScreen.do.clickPaymentMethod("Bank");
-        PaymentScreen.do.clickValidate();
-        TipScreen.check.isShown();
-        Chrome.do.clickMenuButton();
-        Chrome.do.clickTicketButton();
-        TicketScreen.do.clickNewTicket();
-        // order 4
-        ProductScreen.exec.addOrderline("Coca-Cola", "4", "2");
-        ProductScreen.check.totalAmountIs("8.0");
-        Chrome.do.backToFloor();
-        FloorScreen.check.orderCountSyncedInTableIs("5", "4");
-        Chrome.do.clickMenuButton();
-        Chrome.do.clickTicketButton();
-        TicketScreen.check.nthRowContains("4", "Tipping");
+            // Tip 20% on order1
+            TicketScreen.selectOrder("-0001"),
+            TicketScreen.loadSelectedOrder(),
+            TipScreen.isShown(),
+            TipScreen.totalAmountIs("2.0"),
+            TipScreen.percentAmountIs("15%", "0.30"),
+            TipScreen.percentAmountIs("20%", "0.40"),
+            TipScreen.percentAmountIs("25%", "0.50"),
+            TipScreen.clickPercentTip("20%"),
+            TipScreen.inputAmountIs("0.40"),
+            FloorScreen.backToFloor(),
+            FloorScreen.isShown(),
+            Chrome.clickMenuButton(),
+            Chrome.clickTicketButton(),
 
-        // Tip 20% on order1
-        TicketScreen.do.selectOrder("-0001");
-        TicketScreen.do.loadSelectedOrder();
-        TipScreen.check.isShown();
-        TipScreen.check.totalAmountIs("2.0");
-        TipScreen.check.percentAmountIs("15%", "0.30");
-        TipScreen.check.percentAmountIs("20%", "0.40");
-        TipScreen.check.percentAmountIs("25%", "0.50");
-        TipScreen.do.clickPercentTip("20%");
-        TipScreen.check.inputAmountIs("0.40");
-        Chrome.do.backToFloor();
-        FloorScreen.check.isShown();
-        Chrome.do.clickMenuButton();
-        Chrome.do.clickTicketButton();
+            // Tip 25% on order3
+            TicketScreen.selectOrder("-0003"),
+            TicketScreen.loadSelectedOrder(),
+            TipScreen.isShown(),
+            TipScreen.totalAmountIs("6.0"),
+            TipScreen.percentAmountIs("15%", "0.90"),
+            TipScreen.percentAmountIs("20%", "1.20"),
+            TipScreen.percentAmountIs("25%", "1.50"),
+            TipScreen.clickPercentTip("25%"),
+            TipScreen.inputAmountIs("1.50"),
```

Comment:
what happened here, why change the steps?

Reply:
what's the change?